src/Navbar.tsx

```diff
@@ -345,14 +346,32 @@ class Navbar extends React.Component<any, State> {
               // if theres less than 7 navbar items show the home page button
               routesConfig.filter((el) => !el.hideRouteFromNavbar).length <
                 7 && (
-                <NavLink
-                  key={'Home'}
-                  className={`top-nav-button nav-button-container center-content ${isHomeSelectedCssClassName} ${this.getBorder(
-                    '',
-                  )}`}
-                  to={'/'}
-                  text={'Home'}
-                />
+                <Dropdown className={this.getBorder('')}>
+                  <Dropdown.Toggle
+                    variant="light"
+                    id={'Navbar-dropdown-Home'}
+                    className={`nav-button-container top-nav-button ${isHomeSelectedCssClassName}`}                    
+                  >
+                    {/* Clicking the Home item immediately brings you to the Home page (and opens the dropdown, if there are items) */}
+                    <NavLink to='/' text='Home' />
```

Comment:
Clicking on the Home dropdown button still takes you to the home page (but also pops open the dropdown menu, so you can jump to the section that you're interested in).